.github/pull_request_template.md

```diff
@@ -2,7 +2,11 @@
 _Describe the problem or feature in addition to a link to the issues._
 
 ## Approach
-_How does this change address the problem?_
```

Comment:
while we're at it maybe we can add a section for FE screenshots

Reply:
+1